src/app/pcasts/dao/listening_histories_dao.py

```diff
@@ -74,3 +85,16 @@ def get_listening_history(user, max_hs, offset):
     lh.episode = episode_id_to_episode[lh.episode_id]
 
   return listening_histories
+
+def update_listening_history_dismissed(episode_id, user_id, value):
+  assert value is True or value is False
```

Comment:
I think this just throws a nondescriptive assertion error without saying what happened